modules/standard/CTypes.chpl

```diff
@@ -882,10 +911,109 @@ module CTypes {
     return c_addrOf(s);
   }
 
+  /*
+    Returns a :type:`c_ptr` to the underlying buffer of a :type:`~Bytes.bytes`
+
+    Note that the existence of this ``c_ptr`` has no impact on the lifetime of
```

Comment:
same as with other *note* above

Reply:
This is one of the overloads no-doc'd in this PR so I don't think we want to `note` it, but let me know if you think something needs to be added to the other note for this specifically.